.github/workflows/yaml-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint Yaml
+
+on:
+  push:
```

Comment:
should be `pull_request` i think, if a commit doesn't have yml files but the PR in general does.

Reply:
pull_request should also work with forks, since we don't need secrets or write permissions to run validation